core/src/main/java/io/smallrye/stork/Stork.java

```diff
@@ -264,7 +264,16 @@ public static Stork getInstance() {
      * Closes the stork instance.
      */
     public static void shutdown() {
-        REFERENCE.set(null);
+        var previous = REFERENCE.getAndSet(null);
+        if (previous != null) {
+            previous.clear();
+        }
+        //        InMemoryAddressesBackend.clearAll();
```

Comment:
I'm wondering why I kept that code commented. Honestly, I don't remember. Maybe something to cleanup.